test/Microsoft.AspNetCore.WebSockets.Server.Test/AutobahnTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Microsoft.AspNetCore.WebSockets.Server.Test.Autobahn;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.PlatformAbstractions;
+using Xunit;
+
+namespace Microsoft.AspNetCore.WebSockets.Server.Test
+{
+    public class AutobahnTests
+    {
+        // Skip if wstest is not installed for now, see https://github.com/aspnet/WebSockets/issues/95
+        // We will enable Wstest on every build once we've gotten the necessary infrastructure sorted out :).
+        [ConditionalFact]
+        [SkipIfWsTestNotPresent]
+        public async Task AutobahnTestSuite()
+        {
+            var outDir = Path.Combine(PlatformServices.Default.Application.ApplicationBasePath, "autobahnreports").Replace("\\", "\\\\");
+            if (Directory.Exists(outDir))
+            {
+                Directory.Delete(outDir, recursive: true);
+            }
+
+            var spec = new AutobahnSpec(outDir)
+                .IncludeCase("*")
+                .ExcludeCase("12.*", "13.*");
+            var loggerFactory = new LoggerFactory().AddConsole();
+
+            AutobahnResult result;
+            using (var tester = new AutobahnTester(loggerFactory, spec))
+            {
+                await tester.DeployTestAndAddToSpec(ServerType.Kestrel, ssl: false, environment: "ManagedSockets", expectationConfig: expect => expect
+                    .NonStrict("6.4.3", "6.4.4")); // https://github.com/aspnet/WebSockets/issues/99
+
+                await tester.DeployTestAndAddToSpec(ServerType.Kestrel, ssl: true, environment: "ManagedSockets", expectationConfig: expect => expect
+                    .NonStrict("6.4.3", "6.4.4")); // https://github.com/aspnet/WebSockets/issues/99
+
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    // IIS Express tests are a bit flaky, some tests fail occasionally or get non-strict passes
+                    await tester.DeployTestAndAddToSpec(ServerType.IISExpress, ssl: false, environment: "ManagedSockets", expectationConfig: expect => expect
```

Comment:
Conditional based on if IIS Express 10 is installed? Or at least an explicit failure message for the dependency?